### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by NabeelAhsen**
- Remove ENV['SHOPIFY_DOMAIN'] parameter
- Remove unnecessary guides on the README.md
- Change the format of this README to be more of a "quick start" guide
- Remove beta disclaimer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,23 @@
...
 ```
 
-> __Note:__ If you do not have a Shopify API Key or Shopify API Secret, see the following sections of the [Build a Shopify App with Node and React][7] guide:
-> 1. [Expose your dev environment][9]
-> 2. [Get a Shopify API Key and Shopify API secret key][10]
-> 3. [Add the Shopify API Key and Shopify API secret key][11]
+> __Note:__ If you do not have a Shopify API Key or Shopify API Secret, see the following sections of the [Build a Shopify App with Node and React](https://shopify.dev/tutorials/build-a-shopify-app-with-node-and-react/embed-your-app-in-shopify#get-a-shopify-api-key) guide:
+> 1. [Expose your dev environment](https://shopify.dev/tutorials/build-a-shopify-app-with-node-and-react/embed-your-app-in-shopify#expose-your-dev-environment)
+> 2. [Get a Shopify API Key and Shopify API secret key](https://shopify.dev/tutorials/build-a-shopify-app-with-node-and-react/embed-your-app-in-shopify#get-a-shopify-api-key)
+> 3. [Add the Shopify API Key and Shopify API secret key](https://shopify.dev/tutorials/build-a-shopify-app-with-node-and-react/embed-your-app-in-shopify#add-the-shopify-api-key)
```
Comment (rezaansyed) [inline README.md:20]: This link points to an example that names the secret environment variable `SHOPIFY_API_SECRET_KEY` and not `SHOPIFY_API_SECRET`. Can you add a note that accounts for this?

Comment (NabeelAhsen) [reply README.md:20]: Oh right, that's the source of confusion for this 🤦 

Adding a note!

**Review by rezaansyed — APPROVED** [review]
Added a minor comment. But it looks good to me especially given that we point to the official tutorial.

File: README.md (hunk 1 above)
Comment (NabeelAhsen) [reply README.md:23]: See latest update.

**Pull request merged**